JitsiMediaDevices.js

```diff
@@ -73,6 +74,13 @@ const JitsiMediaDevices = {
     },
 
     /**
+     * Returns true if it is possible to have multiple audio input streams.
```

Comment:
This documentation string could use a bit of an improvement. I guess what we mean here is *simultaneous* input devices open, right?